features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml

```diff
@@ -1103,6 +1103,27 @@
             </Read>
         </Scopes>
     </APIResourceCollection>
+    <APIResourceCollection name="org_loginAndRegistration" displayName="Login And Registration" type="organization">
+        <Scopes>
+            <Feature>
+                <Scope name="console:org:loginAndRegistration"/>
+                <Scope name="console:org:loginAndRegistration_view"/>
+                <Scope name="console:org:loginAndRegistration_edit"/>
+            </Feature>
+            <Update>
+                <Scope name="internal_org_governance_update"/>
+            </Update>
+            <Delete/>
+            <Create/>
+            <Read>
+                <Scope name="internal_org_governance_view"/>
+                <Scope name="internal_org_validation_rule_mgt_update"/>
+                <Scope name="internal_org_config_update"/>
```

Comment:
why do we need these update scopes for Read view ?

Can see that those are already used to Read view of the root org view

Reply:
Updated with [5079a1d](https://github.com/wso2/carbon-identity-framework/pull/6976/commits/5079a1d20a3675b2e0131e584e90066a5c06ba62)